.ci/test-docs.sh

```diff
@@ -27,14 +27,23 @@ done
 npm run test
 
 cd ${here}/../doc/7/getting-started/.vuejs
-npm ci
+npm install
```

Comment:
I forgot to ask a question about that point.
npm ci install packages using package-lock.json right ? 
In the getting started (react, vuejs, react-native) kuzzle-sdk is installed as dependency and is overwrite then with postinstall rule by the built kuzzl-sdk of the current sdk repository.

I wasn't sure if npm ci will install the kuzzle-sdk dependency only or if it will also do the postinstall rule...

Reply:
IDK

Reply:
It does the `postinstall`